sli/slidata.cc

```diff
@@ -1383,7 +1383,7 @@ Cvd_sFunction::execute( SLIInterpreter* i ) const
 
   StringDatum* obj = dynamic_cast< StringDatum* >( i->OStack.top().datum() );
   assert( obj );
-  Token t( new DoubleDatum( std::atof( obj->c_str() ) ) );
+  Token t( new DoubleDatum( std::strtod( obj->c_str(), nullptr ) ) );  
```

Comment:
Just a tiny fix to make our code format checker happy:
```suggestion
  Token t( new DoubleDatum( std::strtod( obj->c_str(), nullptr ) ) );
```